lib/govuk_design_system_formbuilder/elements/label.rb

```diff
@@ -61,6 +61,10 @@ def label_options
         }
       end
 
+      def caption_html
```

Comment:
I think to be consistent with [other similar methods](https://github.com/DFE-Digital/govuk_design_system_formbuilder/blob/master/lib/govuk_design_system_formbuilder/containers/fieldset.rb#L40) this could just be `caption`

Reply:
Yep! I will change it.